crates/services/executor/src/executor.rs

```diff
@@ -1508,13 +1507,12 @@ where
         // This is because variable or transfer outputs won't have any value
         // if there's a revert or panic and shouldn't be added to the utxo set.
         if *amount > Word::MIN {
-            let coin = CompressedCoin {
-                owner: *to,
-                amount: *amount,
-                asset_id: *asset_id,
-                maturity: 0u32.into(),
-                tx_pointer: TxPointer::new(block_height, tx_idx),
-            };
+            let mut coin = CompressedCoin::default();
```

Comment:
We want to use explicitly `V1` here. Plus CI fails=) 
<img width="1033" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/4ec50635-84a8-49bc-b821-e967b2999a38">

Reply:
Oops, missed this!